src/igvfd/searches/configs/in_vitro_system.py

```diff
@@ -72,16 +135,62 @@ def in_vitro_system():
             {
                 'title': 'Sample',
                 'facet_fields': [
+                    'taxa',
+                    'classifications',
                     'sample_terms.term_name',
                     'disease_terms.term_name',
+                    'biosample_qualifiers',
+                    'targeted_sample_term.term_name',
+                    'growth_medium',
+                    'embryonic',
+                    'virtual',
+                ]
+            },
+            {
+                'title': 'File Set',
+                'facet_fields': [
+                    'file_sets.@type',
+                    'file_sets.preferred_assay_title',
+                    'file_sets.file_set_type',
+                ]
+            },
+            {
+                'title': 'Construct',
+                'facet_fields': [
+                    'construct_library_sets.file_set_type',
+                    'construct_library_sets.associated_phenotypes.term_name',
+                    'nucleic_acid_delivery',
+                ]
+            },
+            {
+                'title': 'Treatment',
+                'facet_fields': [
+                    'treatments.treatment_type',
                     'treatments.treatment_term_name',
-                    'taxa',
-                    'sex',
-                    'classifications',
+                    'treatments.purpose',
+                    'treatments.depletion',
+                ]
+            },
+            {
+                'title': 'Biomarker',
+                'facet_fields': [
+                    'biomarkers.name',
                     'biomarkers.classification',
-                    'virtual',
-                    'file_sets.assay_term.term_name',
-                    'growth_medium',
+                    'biomarkers.quantification',
+                    'biomarkers.gene.symbol',
+                ]
+            },
+            {
+                'title': 'Modification',
+                'facet_fields': [
+                    'modifications.@type',
+                    'modifications.modality',
+                    'modifications.cas',
+                    'modifications.cas_species',
```

Comment:
Remove cas_species

Reply:
I will keep cas_species, I just forgot it in the google doc.